database/bindata/bindata.go

```diff
@@ -0,0 +1,265 @@
+package bindata
```

Comment:
I think this file should be removed. I creates merge conflicts, churn and can be easily regenerated. We went the same way on `herokuish` https://github.com/gliderlabs/herokuish/pull/11